### DISCUSSION TITLE
Top links of the Atmosphere

### DISCUSSION THREAD

**Discussion by erlend-sh** [likeandscribe/frontpage#167] [Ideas]
What if there was an alternative feed of links, which instead of listing manually added links, it listed the top 10 links on *all of the Atmosphere* in the last 48hrs?

This would essentially be [cross-posting](https://unravel.fyi/blog/3kvlt77tctu2e-frontpage-rfc-cross-posting-to-bluesky) in reverse.

Roughly like this:

1. Find top 10 most liked posts with link(s) on Bluesky et.al. posted within the last 48hrs
2. Cross-post those links on Frontpage

Wouldn’t have to implement the full cross-post RFC to begin with; just seeing the network-wide top links would be really cool for a start.

**Comment by tom-sherman** [comment]
I've been thinking about something similar: after a link is posted to Frontpage, show Bluesky threads alongside the frontpage conversation. Essentially retro-actively enable cross posting for those threads.

**Comment by WillCorrigan** [comment]
This might work well when we introduce communities. Could be an automated "top of bsky today" community that injests these automatically.

It kind of goes against the philosophy of upvotes though unless you get a big collection of them that people can vote on. Although I suppose different communities may want to function differently and just having a daily top news like community where people comment on posts would is fine.

**Reply by codegod100** [reply]
>This might work well when we introduce communities.

I was thinking of requesting a communities features, glad ya'll are on it

**Comment by erlend-sh** [comment]
I changed the title from ‘Top links of Bluesky’ to ‘Top links of the Atmosphere’ to reflect the more diverse state of 2026. Now we have links-specialized services like [Semble](https://semble.so) and [Margin](https://margin.at) to draw from in addition to the *sky network.

We can imagine a new breed of link aggregation app that works more akin [Metacritic](https://en.wikipedia.org/wiki/Metacritic), combining different link-signals from across the at-net into a meta-score.

Semble in particular might be the richest API to draw from, as it specifically concerns itself with cross-network interlinking. I’m sure a collab between frontpage, Semble and [atproto.science](https://atproto.science) would result in a stellar links.atproto.science feed.
